packages/projection/src/chain-sync/ogmios/getNetworkInfo.ts

```diff
@@ -0,0 +1,30 @@
+// Tested in packages/e2e/test/ogmios
+import { ConnectionConfig, createInteractionContext, createStateQueryClient } from '@cardano-ogmios/client';
+import { WithNetworkInfo } from '../../operators';
+import { ogmiosToCore } from '@cardano-sdk/ogmios';
+
+export const getNetworkInfo = async (connectionConfig: ConnectionConfig): Promise<WithNetworkInfo> => {
+  let done = false;
+  const client = await createStateQueryClient(
```

Comment:
Any reason to not use the [CardanoNode](https://github.com/input-output-hk/cardano-js-sdk/blob/e910b2310320418386f692f590d34416ee53445f/packages/core/src/CardanoNode/types/CardanoNode.ts#L33-L55) interface?

Reply:
This is gone now, using [ObservableCardanoNode](https://github.com/input-output-hk/cardano-js-sdk/pull/593/files#diff-2bc9b1dc745118b9f203c03579483b6437a02054833d620145bb14830b9c1fdcR46) in place of this